OuterWildsRandomSpeedrun/SpawnPointMenu/SpawnPointMenuOption.cs

```diff
@@ -0,0 +1,47 @@
+using SpawnPointSelector;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+using OWML.Common;
+using OWML.ModHelper;
+
+namespace OuterWildsRandomSpeedrun
+{
+  public class SpawnPointMenuOption : MenuOption
+  {
+    public IModHelper ModHelper;
+    private static Color OW_ORANGE_COLOR = new Color(0.968f, 0.498f, 0.207f);

```

Comment:
should we move this somewhere more globally available since we have another definition for it elsewhere

Reply:
Sure; I'll do this as a follow-up since we have other PRs in progress that also mess with this variable and I don't want to create a conflict.

Reply:
makes sense